katsdpcal/test/test_control.py

```diff
@@ -299,6 +302,13 @@ def _get_output_stream(self, thread_pool, hostname, port, config,
         self.output_streams[key] = stream
         return stream
 
+    def init_itemgroup(self):
```

Comment:
```suggestion
    def init_item_group(self):
```
Because it's an "ItemGroup" not an "Itemgroup".

Reply:
Done.